_docs/style/typography.md

```diff
@@ -181,6 +181,8 @@ Captions apply to images and visualizations. They should be used to clarify imag
 <div class="hxRow" markdown="1">
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
+<a name="link"></a>
```

Comment:
What's this for? If you're trying to add an anchor tag to the heading, the heading already has one that is added programmatically.

Reply:
I am link from simple buttons to a specific section of the typography document. Is this not the correct way to do it?

Reply:
Ah. So you already have a named anchor set up through the heading. What I mean by that is Jekyll (the system that builds the site) will add an anchor when it adds heading tags (those `h1` through `h6` tags that indicate a heading) when creating the HTML from the markdown. So, when you link to a specific section from another page, you only need to create the link on the other page using the square brackets and parentheses (e.g., in this case, `[link text]({{site.baseurl}}/style/typography.md#link)` if you want to use the system you used before) and can leave this page as is.

Reply:
ah! Ok, so that anchor tag is redundant. I removed it and it works just fine.